deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml

```diff
@@ -4322,7 +4322,6 @@ metadata:
   namespace: kube-system
   labels:
     clickhouse.altinity.com/chop: 0.25.6
-
```

Comment:
deploy/opreator files are generated. So please remove from the PR

Reply:
actually files updated after run generation scripts